saleor/graphql/schema.graphql

```diff
@@ -15905,6 +15905,16 @@ type Mutation {
     input: GiftCardSettingsUpdateInput!
   ): GiftCardSettingsUpdate @doc(category: "Gift cards")
 
+  """
+  Reset circuit breaker for provided app. 
+  
+  Requires one of the following permissions: MANAGE_SETTINGS.
+  """
+  resetCircuitBreaker(
```

Comment:
I don't think we should be using "circuit breaker" in our API naming. It's a design pattern, we don't have objects named "singleton," etc.

Reply:
@tomaszszymanski129

I see this conversation is already resolved but there is no comment regarding what's the resolution here.

Are we renaming this with a different name?

Reply:
Indeed, something weird happened to the schema, though for now I renamed it to `ReenableSyncWebhooks`, though we might change it later if we find a better name, it's a quick refactor.